docs/using-the-jdbc-driver/UsingTheJdbcDriver.md

```diff
@@ -200,8 +200,8 @@ The AWS JDBC Driver has several built-in plugins that are available to use. Plea
-| [Federated Authentication Plugin](./using-plugins/UsingTheFederatedAuthPlugin.md)                                 | `federatedAuth`           | Aurora, RDS[^2]                 | Enables users to authenticate using Federated Identity and then connect to their Amazon Aurora Cluster using AWS Identity and Access Management (IAM).                                                                                                                                                                                                                                                                                                                                                                 |                                                                                                                                                                                                               |
-| [Okta Authentication Plugin](./using-plugins/UsingTheOktaAuthPlugin.md)                                           | `okta`                    | Aurora, RDS[^2]                 | Enables users to authenticate using Federated Identity and then connect to their Amazon Aurora Cluster using AWS Identity and Access Management (IAM).                                                                                                                                                                                                                                                                                                                                                                 |                                                                                                                                                                                                               |
+| [Federated Authentication Plugin](./using-plugins/UsingTheFederatedAuthPlugin.md)                                 | `federatedAuth`           | Aurora, RDS[^2]                 | Enables users to authenticate using Federated Identity and then connect to their Amazon Aurora Cluster using AWS Identity and Access Management (IAM).                                                                                                                                                                                                                                                                                                                                                                 | [Jackson Databind](https://central.sonatype.com/artifact/com.fasterxml.jackson.core/jackson-databind)                                                                                                         |
```

Comment:
Can you please also update the pre-req sections in the relevant docs? e.g. https://github.com/aws/aws-advanced-jdbc-wrapper/blob/main/docs/using-the-jdbc-driver/using-plugins/UsingTheFederatedAuthPlugin.md
We should also mention:
```
[AWS Java SDK RDS v2.7.x](https://central.sonatype.com/artifact/software.amazon.awssdk/rds)
[AWS Java SDK STS v2.7.x](https://central.sonatype.com/artifact/software.amazon.awssdk/sts)
```
And do we need JSOUP?